.github/ISSUE_TEMPLATE.md

```diff
@@ -19,13 +12,9 @@ Please provide any relevant information about your setup. This is important in c
 * OpenMined Unity App Version:
 
 
-## Expected Behavior
-
-Please describe the behavior you are expecting
-
-## Current Behavior
+## Description
 
-What is the current behavior?
+Please describe the behavior you are expecting and the current behavior?
 
 ## Failure Information (for bugs)
```

Comment:
This should probably be a separate issue/PR, but we should probably have two separate issue templates for bugs and features (since those are by far the two most common issue types).  Would you be able to open up an issue for this?

Reply:
Sure, I'll open an issue for this and perhaps work on it myself.